README.md

```diff
@@ -19,52 +19,66 @@ Useful links
 
 Quick Start Guide
 --------------
+This guide allows you to build salmiac from source and convert your docker application into a one that can run in a nitro enclave. 
 
-1. Install Rust:
-   Follow [this](https://www.rust-lang.org/tools/install) guide.
+1. Set up your Ubuntu based build system:
+    - Install Rust:
```

Comment:
Do we need to specify what version of the rust toolchain is required?

Reply:
The build-converter.sh script adds the toolchain targets it needs with the help of rustup.